httpd/handler.go

```diff
@@ -175,6 +179,127 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 	httpResults(w, results, pretty)
 }
 
+type Point struct {
```

Comment:
time.Time is a special case, and they are actually fixing this in a future release of Go.  One trick is you can make it a pointer to time, but I've never been a big fan of that personally.  Like I said, this was more food for thought than an action item.  If it's working for now, I would leave it and we can refactor at a future date.